source/code/scxcorelib/pal/scxlibglob.cpp

```diff
@@ -22,12 +22,12 @@ namespace SCXCoreLib
         // if directory paths are non-existant, provide default system library paths
         if (paths.size() == 0)
         {
-            m_paths.push_back(L"/lib");
-            m_paths.push_back(L"/usr/lib");
 #if PF_WIDTH == 64
             m_paths.push_back(L"/lib64");
             m_paths.push_back(L"/usr/lib64");
```

Comment:
Yeah there's a bunch of other library directories used across all our Linux platforms (as we've learned from openssl library locations).  For now this change is good enough, since there's only one consumer of SCXLibGlob that I know of (installedsoftwaredepend.cpp, which uses SCXLibGlob to find libpopt and librpm on RPM systems).  If we end up wanting to include a bunch of other system library paths, we can do that at a later time in a different commit.